frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -28,7 +28,6 @@ export const scene: SceneExport = {
 
 export function SurveyComponent({ id }: { id?: string } = {}): JSX.Element {
     const { isEditingSurvey, surveyMissing } = useValues(surveyLogic)
-    const showSurveyForm = id === 'new' || isEditingSurvey
```

Comment:
I check this state in the `urlToAction` handler instead now, so we don't need to evaluate this logic in the component.